docs/advanced/custom_readers.md

```diff
@@ -0,0 +1,155 @@
+
+rio-tiler provides multiple base classes from which it derives its main readers: [COGReader](/readers/#cogreader) and [STACReader](/readers/#stacreader). The base classes can be used by users to built custom readers (see [rio-tiler-pds](https://github.com/cogeotiff/rio-tiler-pds)).
```

Comment:
```suggestion
`rio-tiler` provides multiple base classes from which it derives its main readers: [`COGReader`](/readers/#cogreader) and [`STACReader`](/readers/#stacreader). The base classes can be used to build custom readers (see [`rio-tiler-pds`](https://github.com/cogeotiff/rio-tiler-pds)).
```

Reply:
I'm not gonna take the time now to do such stylistic changes for the whole PR, I think having a consistent style is good because it gives off a professional feel, but it takes a bit of time reading through the docs